ccmlib/node.py

```diff
@@ -275,21 +275,21 @@ def show(self, only_status=False, show_cluster=True):
         """
         self.__update_status()
         indent = ''.join([" " for i in xrange(0, len(self.name) + 2)])
-        print_("%s: %s" % (self.name, self.__get_status_string()))
+        print_("{}: {}".format(self.name, self.__get_status_string()))
         if not only_status:
             if show_cluster:
-                print_("%s%s=%s" % (indent, 'cluster', self.cluster.name))
-            print_("%s%s=%s" % (indent, 'auto_bootstrap', self.auto_bootstrap))
-            print_("%s%s=%s" % (indent, 'thrift', self.network_interfaces['thrift']))
+                print_("{}{}={}".format(indent, 'cluster', self.cluster.name))
+            print_("{}{}={}".format(indent, 'auto_bootstrap', self.auto_bootstrap))
+            print_("{}{}={}".format(indent, 'thrift', self.network_interfaces['thrift']))
             if self.network_interfaces['binary'] is not None:
-                print_("%s%s=%s" % (indent, 'binary', self.network_interfaces['binary']))
-            print_("%s%s=%s" % (indent, 'storage', self.network_interfaces['storage']))
-            print_("%s%s=%s" % (indent, 'jmx_port', self.jmx_port))
-            print_("%s%s=%s" % (indent, 'remote_debug_port', self.remote_debug_port))
-            print_("%s%s=%s" % (indent, 'byteman_port', self.byteman_port))
-            print_("%s%s=%s" % (indent, 'initial_token', self.initial_token))
+                print_("{}{}={}".format(indent, 'binary', self.network_interfaces['binary']))
+            print_("{}{}={}".format(indent, 'storage', self.network_interfaces['storage']))
+            print_("{}{}={}".format(indent, 'jmx_port', self.jmx_port))
+            print_("{}{}={}".format(indent, 'remote_debug_port', self.remote_debug_port))
+            print_("{}{}={}".format(indent, 'byteman_port', self.byteman_port))
+            print_("{}{}={}".format(indent, 'initial_token', self.initial_token))
```

Comment:
If @mambocab were here, he'd suggest a clever way of reducing all this code dupe, which I won't force on you. But I ping him here to see if he agrees that I should do it